x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts

```diff
@@ -145,6 +146,18 @@ export const createStartServicesMock = (
     return { pause: true, value: 1000 };
   });
   data.query.timefilter.timefilter.calculateBounds = jest.fn(calculateBounds);
+
+  timelineDataService.query.timefilter.timefilter.getAbsoluteTime = jest.fn(() => ({
```

Comment:
@michaelolo24 I guess you added these mocks to the regular data service to make the tests pass. Do you remember which tests those were? I have a feeling, we only need to add the mocks to the `timelineDataService` now.

Reply:
👍  I personally prefer only mocking what's needed to be mocked so there are less surprises. (https://github.com/elastic/kibana/pull/180850/commits/f5fd1ce2cf5d7707c50956ae86f20f2f211ea38c)